ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md

```diff
@@ -0,0 +1,119 @@
+
+**Quality Engineer**
+
+Service development has a life-cycle that ultimately results in a service which is deployed and used by customers. The quality phase of that life-cycle is absolutely critical in the reliability of that service, as SREs need to be able to trust what is being deployed. That is the role for this persona. It is important to note, that this is not the role of the SRE, especially testing in Production environments.
+
+**Site Reliability Engineer (SRE)**
+
+The Site or Service Reliability Engineer is an essential part of the service. Without the SRE, the service is incomplete. Their primary focus is ensuring that the services are "running". When there is a problem, it is up to them to resolve the issue as quickly as possible. However, this is only the end goal. The role of an SRE is to contribute to the reliability of the service so that the need to fix something is the exception rather than the rule. This is best epitomized in this quote by one of Red Hat's SRE Managers: "Do not accept repeated failure". That is, if something fails (e.g. SRE is alerted), then follow through with all "actions" so that the problem does not repeat. These actions can be things like: from identifying the issue with the responsible engineering team through to actually fixing the problem in code, personally. 
+
+In many organizations, SRE functions support more than one service. Many projects undertaken by SRE teams provide the tooling, workflows and platforms needed to operate at scale. A key metric is for the SRE team to grow sub-linearly with "hockey stick" customer growth while minimizing toil and technical debt.
+
+SREs are also valuable consultants, particularly in these key areas: observability; automation; resilience; toil prevention; capacity planning; and, operability. They are most familiar with their operational boundaries, and can best articulate the problems being encountered since they are hands-on.
+
+**Support Agent**
+
+This persona provides assistance to Service Consumers, usually in the form of support cases raised by customers. They may require elevated levels of access to the service internals in order to help consumers, but in a way that ensures they cannot degrade the service itself. 
+
+**Exec (VP+)**
+
+While the Service Owner has a direct interest in the commercial success of the service, they are empowered and enabled by the Executive arm of the organization. The Service Owner has an interest in the success of the service. The executive leadership has an interest in the success of the company as a whole, of which that service is part of. Ultimately reporting to the board, the executive implements the  strategies that are designed to meet the goals of the company.
```

Comment:
This is a bunch of opening with "have we mentioned Service Owners?  They are different from Execs" ;).  Can we rephrase to say something about execs first?  Maybe:

> Executive leadership sets direction and policy for the organization as a whole... blah, blah important stuff like people policy and culture and working with the board, blah, blah.  They work with Service Owners to allocate available resources between services and manage service lifecycle, creating new services, evolving and refocusing existing services, and sun-setting or restructuring less productive services.

or some such?

Reply:
@wking  are you saying that the "Execs" persona should appear before the "Service Owners"?

Reply:
No, I'm saying that in the section on execs, we should begin by describing execs. You currently spend the first two sentences of the execs section describing service owners.

Reply:
@wking That was to attempt to contrast between execs and service owners. How would you word it?

Reply:
@wking As per our offline chat, I have re-worded to: `Ultimately reporting to the board, the executive is responsible for implementing the strategies that are designed to meet the goals of the company. Therefore, the executive leadership has an indirect interest in the success of the service, insofar as it is reinforces the company goals. While the Service Owner has a direct interest in the commercial success of the service, they are empowered and enabled by the Executive arm of the organization.`